packages/imt.sol/test/LazyIMT.ts

```diff
@@ -2,10 +2,26 @@ import { expect } from "chai"
 import { run } from "hardhat"
 import { poseidon2 } from "poseidon-lite"
 import { IMT } from "@zk-kit/imt"
+import type { BigNumber } from "ethers"
 import { LazyIMT, LazyIMTTest } from "../typechain-types"
 
 const random = () => poseidon2([Math.floor(Math.random() * 2 ** 40), 0])
 
+// Given a a merkle proof (elements and indexes) and a leaf, calculates the root
+function calculateRoot(element: BigInt, proofElements: BigNumber[], proofIndexes: boolean[]) {
```

Comment:
This function should be inlined in the `Should produce valid Merke proofs for different trees` test because it is not used anywhere else.

Reply:
right, moved inside: 9e27c98ef556e3bf4c30e7ae716d126f274cd620